src/js/components/ClickToSelect.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class ClickToSelect extends React.Component {
+  constructor() {
+    super();
+    this.selectAll = this.selectAll.bind(this);
+  }
+
+  selectAll() {
+    document.getSelection().selectAllChildren(this.refs.node);
```

Comment:
did you ensure that `selectAllChildren` works on all browsers?

https://developer.mozilla.org/en-US/docs/Web/API/Selection/selectAllChildren

i suppose if it doesn't, then it simply won't select and it will be ok